scripts/cat/cats.py

```diff
@@ -565,38 +565,24 @@ def die(self, body: bool = True):
             self.illnesses.clear()
 
         # Deal with leader death
-        text = ""
         darkforest = game.clan.instructor.df
         isoutside = self.outside
         if self.status == "leader":
-            if game.clan.leader_lives > 0:
-                lives_left = game.clan.leader_lives
-                death_thought = Thoughts.leader_death_thought(
-                    self, lives_left, darkforest
-                )
-                final_thought = event_text_adjust(self, death_thought, main_cat=self)
-                self.thought = final_thought
-                return ""
-            elif game.clan.leader_lives <= 0:
+            if game.clan.leader_lives <= 0:
```

Comment:
I'm not sure this change won't have unanticipated problems elsewhere in the code, even if it makes sense. There was a change in event code, but I don't recall seeing anything for patrols.

Reply:
![image](https://github.com/user-attachments/assets/654679fb-d63f-4b6b-a096-e539a147ea31)
I redid my refactoring step by step and I feel confident it is working the same way. I might be wrong though